projects/v2/src/app/app-config.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable }  from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { SheetService } from './services/sheet.service';
```

Comment:
Not sure why this keeps getting included. If you want to use the sheetService from another component, inject the sheetService there.